tests/autoclassify/test_detect_intermittents.py

```diff
@@ -26,8 +26,7 @@ def test_detect_intermittents(test_job, failure_lines, classified_failures,
     MatcherManager._detector_funcs = {}
     detector = MatcherManager.register_detector(TestFailureDetector)
 
-    call_command('detect_intermittents', retriggered_job.repository.name,
-                 retriggered_job.guid)
+    call_command('detect_intermittents', str(test_job.id))
```

Comment:
Perhaps we should just get rid of the management command entirely?

Or at the least, shouldn't the tests also be switched from `call_command` to triggering a sync celery task too?